.github/workflows/periodic-integration-tests.yml

```diff
@@ -214,42 +214,42 @@ jobs:
 
       - name: Install any data connector packages
         run: |
-          aqueduct install redshift
+          pip3 install boto3
 
       - name: Setup Hosted Data Integrations
         working-directory: scripts/data
         run: python3 setup_hosted.py --aws-key-id ${{ secrets.SAURAV_AWS_ACCESS_KEY_ID }} --aws-secret-key ${{ secrets.SAURAV_AWS_SECRET_ACCESS_KEY }}
 
-      - name: Run the SDK Data Integration Tests
-        working-directory: integration_tests/sdk
-        run: python3 run_tests.py --data-integration -n 2
+      # - name: Run the SDK Data Integration Tests
+      #   working-directory: integration_tests/sdk
+      #   run: python3 run_tests.py --data-integration -n 2
 
-      - uses: ./.github/actions/upload-artifacts
-        if: always()
-        with:
-          prefix: Data Connectors
+      # - uses: ./.github/actions/upload-artifacts
+      #   if: always()
+      #   with:
+      #     prefix: Data Connectors
 
       - name: Teardown Hosted Data Integrations
         if: always()
         working-directory: scripts/data
         run: python3 teardown_hosted.py --aws-key-id ${{ secrets.SAURAV_AWS_ACCESS_KEY_ID }} --aws-secret-key ${{ secrets.SAURAV_AWS_SECRET_ACCESS_KEY }}
 
       # Sets it as an environmental variable.
-      - name: Get the Slack ID for the current oncall
-        if: always()
-        run: |
-          aws s3 cp s3://aqueduct-assets/oncall.yml ./oncall.yml
-          echo "ONCALL_SLACK_MEMBER_ID=$(python3 scripts/get_current_oncall.py --file ./oncall.yml)" >> $GITHUB_ENV
-
-      - name: Report to Slack on Failure
-        if: always()
-        uses: ravsamhq/notify-slack-action@v1
-        with:
-          status: ${{ job.status }}
-          notification_title: ""
-          message_format: "{emoji} *{workflow}* has {status_message}"
-          footer: "{run_url}"
-          notify_when: "failure,warnings"
-          mention_users: ${{ env.ONCALL_SLACK_MEMBER_ID }}
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.ACTION_MONITORING_SLACK }}
+      # - name: Get the Slack ID for the current oncall
```

Comment:
It's probably good for us to set timeouts for both the setup and teardown steps here.